CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v2.1.1 / 2021-07-28
+
+* [CHANGE] Replace multiListWatcher with independent listWatchers per namespace #1499
+* [CHANGE] go.mod: Update and minimize dependencies #1529
+* [BUGFIX] Use BuilderInterface instead of internal/store.Builder in metricshandler #1537
+* [BUGFIX] Add WithAllowLabels to public BuilderInterface #1514
```

Comment:
I would consider both of these PRs changes instead of bug fixes since they are actual improvements made to the builder interface to make it easier for users to extend ksm functionalities.